awx/ui/client/features/projects/projectsList.view.html

```diff
@@ -80,6 +80,10 @@
                         </div>
                         <at-truncate string="{{ project.scm_revision }}" maxLength="7"></at-truncate>
                     </div>
+                    <at-row-item
+                        label-value="{{:: vm.strings.get('list.ROW_ITEM_LABEL_DESCRIPTION')}}"
+                        value="{{ project.description }}">
```

Comment:
@saito-hideki Please apply the `sanitize` filter to the new values here and in the `templatesList` - you can do this by piping the value into `sanitize` like we do here: https://github.com/ansible/awx/blob/devel/awx/ui/client/features/applications/list-applications-users.view.html#L24

Alternatively, you can filter the value in the controller by using `$filter('sanitize')`, like we do here: https://github.com/ansible/awx/blob/devel/awx/ui/client/features/applications/list-applications-users.controller.js#L90

Reply:
@jakemcdermott Thank you for the review! I have modified to use `sanitize` filter.
It would be appreciated if you take a look at this fix.